signup.html

```diff
@@ -37,7 +37,9 @@
           <label for="password">비밀번호</label>
           <div class="password-wrapper">
             <input id="password" name="password" type="password" placeholder="비밀번호를 입력해 주세요" required />
-            <img src="images/login_icon/eye-invisible.svg" alt="비밀번호 감추기" class="toggle-password" />
+            <button>
```

Comment:
❗️ 수정요청
해당 버튼에 type이 명시되어 있지않아, default type인 submit 버튼이 됩니다~
그래서 버튼을 클릭하면 form이 제출될거에요! type을 login.html 파일에서처럼 명시해주세요!

```suggestion
            <button type="button">
```